tests/link/pytorch/test_blockwise.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pytest
+
+import pytensor
+import pytensor.tensor as pt
+from pytensor.tensor.shape import specify_broadcastable
+
+
+torch = pytest.importorskip("torch")
+
+
+def test_blockwise_broadcast():
+    _x = np.random.rand(5, 1, 2, 3)
+    _y = np.random.rand(3, 3, 2)
+
+    x = specify_broadcastable(pt.tensor4("x"), 1)
+    y = pt.tensor3("y")
+
+    f = pytensor.function([x, y], [x @ y], mode="PYTORCH")
```

Comment:
I want to use a custom test core_op (that behaves just like dot) to make sure we will always be testing Blockwise

Reply:
Could you elaborate? I tried this, and it failed because torch doesn't handle `TestOp`

```py
class TestOp(Op):
    def __init__(self):
        self.called = 0

    def make_node(self, *args):
        return Apply(self, list(args), [x.type() for x in args])

    def perform(self, inputs, outputs):
        x, y = inputs
        [z] = outputs
        z[0] = x @ y


def test_blockwise_broadcast():
    _x = np.random.rand(5, 1, 2, 3)
    _y = np.random.rand(3, 3, 2)

    x = pt.tensor4("x", shape=(5, 1, 2, 3))
    y = pt.tensor3("y", shape=(3, 3, 2))
    z = TestOp()(x, y)

    f = pytensor.function([x, y], z, mode="PYTORCH")
    res = f(_x, _y)
    assert tuple(res.shape) == (5, 3, 2, 2)
    np.testing.assert_allclose(res, _x @ _y)
```

```
================================================================= short test summary info =================================================================
FAILED tests/link/pytorch/test_blockwise.py::test_blockwise_broadcast - NotImplementedError: No PyTorch conversion for the given `Op`: TestOp.
```

Reply:
You're missing two things, you need to blockwise for the purposes of the test, and you need to add a torch dispatch to the core TestOp (here on the test file)

Reply:
Got it, I can add the dispatch here then

Reply:
This has been added.